src/controllers/commands/CommandController.cpp

```diff
@@ -31,6 +32,136 @@
             "/followersoff"                                             \
     }
 
+namespace {
+using namespace chatterino;
+
+void sendWhisperMessage(const QString &text)
+{
+    auto app = getApp();
+    app->twitch.server->sendMessage("jtv", text.simplified());

```

Comment:
please add a comment here stating that we should not be sending whispers through jtv, but rather to your own username

Reply:
I don't understand, but okay, but i believe you should be using the chat youre currently writing in, because the error messages get sent on the same channel, thats why I added "jtv" channel to the "global" error messages checks https://github.com/fourtf/chatterino2/blob/e201881516a06f8cd14038ac7e6b32cc034404f8/src/providers/twitch/IrcMessageHandler.cpp#L532

Reply:
Sending whispers to your own channel means you will not be rate-limited, which can happen if you send whispers to jtv

Reply:
https://github.com/fourtf/chatterino2/blob/b7d04a648ff7ab584c07f5d9d58defe89b393253/src/controllers/commands/CommandController.cpp#L42-L43